packages/vapor/scss/typography/elements.scss

```diff
@@ -3,3 +3,36 @@
     font-size: var(--default-font-size);
     line-height: $big-line-height;
 }
+
+.text {
+    --color: var(--default-text-color);
+    color: var(--color);
+
+    &.mod-emphasized {
+        --color: var(--title-text-color);
+    }
+    &.mod-disabled {
+        --color: var(--grey-60);
+    }
+    &.mod-success {
+        --color: var(--success-60);
+    }
+    &.mod-error {
+        --color: var(--critical-70);
+    }
+    &.mod-major {
```

Comment:
major? :eyes:

Reply:
It seems to only be used with `color-dot state-major`, so I took the name from there. However, for other mods, I think `mod-error` will make more sense than `state-critical` since the use cases for text and color dot are different.